code/modules/mob/pulling.dm

```diff
@@ -1,8 +1,10 @@
 //this and stop_pulling really ought to be /mob/living procs
-/mob/proc/start_pulling(atom/movable/AM)
+/mob/proc/start_pulling(atom/movable/AM, state, force = move_force, suppress_message = FALSE)
```

Comment:
Erm. Isn't this file supposed to be deleted?

Reply:
Oof yeah I forgot to delete it after I got pulling working. I had it disabled in the DME, I was actually going to ask if you wanted it deleted or not and forgot to ^-^'